src/awkward/_v2/behaviors/string.py

```diff
@@ -250,7 +254,7 @@ def register(behavior):
     # behavior[ak.nplike.numpy.equal, "bytestring", "bytestring"] = _string_equal
     # behavior[ak.nplike.numpy.equal, "string", "string"] = _string_equal
     # behavior[ak.nplike.numpy.not_equal, "bytestring", "bytestring"] = _string_notequal
-    # behavior[ak.nplike.numpy.not_equal, "string", "string"] = _string_notequal
+    behavior[ak.nplike.numpy.not_equal, "string", "string"] = _string_notequal
```

Comment:
With `_string_equal` implemented, it should be possible to enable all four of these. I guess only one was needed for the test. (Enabling these string and categorical behaviors is one of the cards.)